base/libgit2/libgit2.jl

```diff
@@ -528,7 +528,7 @@ end
 function set_ssl_cert_locations(cert_loc)
     cert_file = isfile(cert_loc) ? cert_loc : Cstring(C_NULL)
     cert_dir  = isdir(cert_loc) ? cert_loc : Cstring(C_NULL)
-    cert_file == C_NULL && cert_dir == C_NULL && return
+    cert_file === C_NULL && cert_dir === C_NULL && return
```

Comment:
This is still wrong. The function also look very strange. Is it actually setting the envs to `"Cstring(0x0000000000000000)"`?

```julia
julia> ENV["AAA"] = Cstring(C_NULL)
Cstring(0x0000000000000000)

julia> ENV["AAA"]
"Cstring(0x0000000000000000)"
```

The setting of the `ENV` should probably just be folded with the `isdir`/`isfile` check (can they possibly both pass?) and clear the corresponding variable if necessary.